app_tests/benchmark_tests/llm/sglang_benchmarks/conftest.py

```diff
@@ -64,39 +64,39 @@ def pre_process_model(request, tmp_path_factory):
 
 
 @pytest.fixture(scope="module")
-def write_config(request, pre_process_model):
+def write_config(request, model_test_dir):
```

Comment:
Need to change the name of the fixture in `conftest.py` from `pre_process_model` to `model_test_dir` as well